infrastructure/w3id/README.md

```diff
@@ -50,6 +50,16 @@ the URL of its “controller”, the owner, then such URL should be resolved to
 
 The Identifier would be loosely bound to a set of keys, meaning an identifier is not derived from a set of keys making it easy to change the keys in case someone's keys gets compromised, or they turn up on the beach naked and need a new set of keys as their device is lost.
 
+### Additional Considerations
+
+#### Friend Based Recovery
+
+2-3 Friends may verify the identity of a person to recover lost metastate ID on the eVault.
+
+#### Migration of eVault
+
+In the implementation it must be ensured that the file is still accessible regardless of the eVault it is stored in.
```

Comment:
```suggestion
In the implementation it must be ensured that the file is still accessible regardless of the eVault it is stored in. This can be done via recording also-known-as records in the register for an eVault each time someone migrates their eVault. For example if a user migrates  evault `e4d909c2-5d2f-4a7d-9473-b34b6c0f1a5a` to `f2a6743e-8d5b-43bc-a9f0-1c7a3b9e90d7`, the register will store an entry pointing all requests to `e4d909c2-5d2f-4a7d-9473-b34b6c0f1a5a` to be redirected to `f2a6743e-8d5b-43bc-a9f0-1c7a3b9e90d7`
```